hedyorg/hedy
# [UI] Implements button to hide / show editor and expand the output panel

**Description**
In this PR we implement a button as an overlay on the code window to enable the user to hide/show the editor and thereby expand the output panel. This makes the blur option redundant and enables the user to get more space for their output.

**Fixes**
This PR fixes #3608

**How to test**
Notice the new button as an overlay on the code editor. Verify that it can be used to hide/show the code editor and thereby shrink / enlarge the output window. Also verify that the button isn't shown in a screen smaller than Tailwind `lg`.
